.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false

```

Comment:
Would you prefer to enable the possibility to add issues without using a template?

Reply:
I'm torn on this.

For bugs and feature requests, this is not neccessary. We don't have some complex JIRA-like form with dozens of fields, just a description + affected version + OS.

However, I think we may sometimes want to create issues as TODO items (e.g. for project boards) and then these two options may not be a good fit. Maybe we should add a new type "Task" for those kind of issues? Or could we restrict blank issues for contributors only?

Reply:
:+1: "TODO" for contributors only

Reply:
> Or could we restrict blank issues for contributors only?

If thats possible, that would be ideal IMO.

If not, we should have blank issues enabled for all IMO.

Reply:
I think that with config version we can't do something like that. I'll enable blank issues in the config.